mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/param/display/Path.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.annotation.param.display;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import org.mule.runtime.extension.api.annotation.param.Parameter;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Marks a {@link Parameter} field as path to a file or directory.
```

Comment:
why just a field? Can't any parameter be like this? (e.g: file:list, file:read, etc)